database/migrations/2025_11_06_000001_update_user_accounts_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    public function up(): void
+    {
+        Schema::table('UserAccounts', function (Blueprint $table) {
+            $table->dropIndex('users_last_activity_id_index');
+
+            $table->dropColumn([
+                'PasswordResetToken',
+                'fbUser',
+                'fbPrefs',
+                'cookie',
+                'LastActivityID',
+            ]);
```

Comment:
* `PasswordResetToken` is being migrated to the `password_reset_tokens` table and now supports multiple tokens per user_id in case emails take an unusual amount of time to get delivered and the user makes more requests. Tokens now expire after 5 days, and there's still a rate limited in effect for requesting tokens.
* `fbUser` and `fbPrefs` are remnants from long ago. It's hard to tell when the feature was eliminated, but the most recently created account that has it set was made in 2018.
* I'm not exactly sure how `cookie` was used, and it's very hard to grep for. I think I got all the references.
* `LastActivityID` points to a table that no longer exists (#2074)